frontend/app/components/input/input.tsx

```diff
@@ -135,6 +136,14 @@ export class Input extends Component<Props, State> {
     });
   }
 
+  async onPaste(e: ClipboardEvent) {
+    if (!(e.clipboardData && e.clipboardData.files.length > 0)) {
+      return;
+    }
+    const files = Array.from(e.clipboardData.files);
```

Comment:
Forgot `e.preventDefault()`. It's still buggy.
![Screenshot 2019-07-16 at 02 16 06](https://user-images.githubusercontent.com/884649/61255223-43f7cd80-a770-11e9-90ca-264fdff94964.png)

Reply:
Which are browser do you use?

Reply:
Firefox 69b4.

Reply:
But I don't think it's browser related. Upon paste I see that it pastes file's name. And uploadImages taking value of input while uploading to insert all this "...uploading" placeholders, so race appears, and I think what causes it is current flow
* image pasted
* uploadImages takes value of input
* event pastes image filename into input
* uploadImages deals with current cursor position and selection which and by this time taken value is out of sync.

Something like that, dunno. We should prevent pasting text data from event.

Reply:
I check it on Mac
- Chrome upload only one file
- Firefox paste to file name
- Safari upload to file

Reply:
I think it rare case and we can skip that.

Reply:
Rare, Firefox users rare? 0_o

Reply:
That user pastes two file

Reply:
Oh, yeah, even github doesn't deal with it, it's fine to skip :-) It's browser's issue.

Reply:
> I check it on Mac
Yeah, same, seems like it's state of api.

Reply:
It  works without `e.preventDefault`. What is sense to make `preventDefault`?

Reply:
I think it could be merged.

Reply:
![image](https://user-images.githubusercontent.com/884649/61280378-6dd8e080-a7c0-11e9-8126-5a34d482c135.png)

It works incorrectly. At least on firefox it leaves strange chunk of filename in input.

`preventDefault` prevents default which is paste of file's name.

Try to paste file with filename `Screenshot 2019-07-16 at 11.56.20.png` on firefox.

Reply:
Hm, It works fine without prevent  in firefox on my laptop, but  if it helps I added it.